frontend/packages/operator-lifecycle-manager/src/components/operand/operand-link.tsx

```diff
@@ -34,6 +44,8 @@ export const OperandLink: React.SFC<OperandLinkProps> = (props) => {
 export type OperandLinkProps = {
   obj: K8sResourceKind;
   csvName?: string;
+  cancel?: () => void;
```

Comment:
Why is the callback for clicking the link called `cancel`? Maybe `onClick` instead?

Reply:
changed to `onClick`